Products/ZenUI3/browser/resources/css/zen-cse.css

```diff
@@ -2255,6 +2255,53 @@ BYE BYE BORDERS
       visibility: hidden;
     }
 
+    /*<ZEN-30423> white theme styles for reports start*/
+    .z-cse.z-cse-reports th.tabletitle div.tabletitlecontainer {
+        background-color: #d2d2d2 !important;
+        vertical-align: middle;
+    }
+    .z-cse.z-cse-reports a.tableheaderselected,
+    .z-cse.z-cse-reports a.tableheader {
+        display: block;
+    }
+    .z-cse.z-cse-reports th.tableheaderselected,
+    .z-cse.z-cse-reports th.tableheader {
+        background-image: url(/++resource++zenui/img/xtheme-zenoss/grid/grid3-hrow.png) !important;
+        background-color: #ccc !important;
```

Comment:
I am wary of any files from the old xtheme-zenoss days.  There are no beveled backgrounds in the new UI.  Flat. Flat. Flat.

Reply:
changed.